types/node/string_decoder.d.ts

```diff
@@ -48,7 +48,7 @@ declare module 'string_decoder' {
          * @since v0.1.99
          * @param buffer A `Buffer`, or `TypedArray`, or `DataView` containing the bytes to decode.
          */
-        write(buffer: Buffer): string;
+        write(buffer: Buffer | Uint8Array | Int8Array | Uint8ClampedArray | Uint16Array | Int16Array | Uint32Array | Int32Array | BigUint64Array | BigInt64Array | Float32Array | Float64Array | DataView): string;
```

Comment:
Just use `NodeJS.ArrayBufferView` here. 

`buffer: Buffer | NodeJS.ArrayBufferView`

Interesting, is it possible to use `ArrayBuffer` here?